qt/applications/workbench/workbench/plugins/workspacewidget.py

```diff
@@ -174,7 +175,7 @@ def _do_slice_viewer(self, names):
         """
         for ws in self._ads.retrieveWorkspaces(names, unrollGroups=True):
             try:
-                presenter = SliceViewer(ws=ws, parent=self)
+                presenter = SliceViewer(ws=ws, parent=self, conf=CONF)
```

Comment:
Good use of dependency injection to separate the configuration from the application.